src/Utopia/Messaging/Adapters/SMS/Twilio.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Utopia\Messaging\Adapters\SMS;
+
+use Utopia\Messaging\Messages\SMS;
+use Utopia\Messaging\Adapters\SMS\SMS as SMSAdapter;
+
+class Twilio extends SMSAdapter
+{
+    /**
+     * @param string $accountSid Twilio Account SID
+     * @param string $authToken Twilio Auth Token
+     */
+    public function __construct(
+        private string $accountSid,
+        private string $authToken
+    ) {
+    }
+
+    public function getName(): string
+    {
+        return 'Twilio';
+    }
+
+    public function getMaxMessagesPerRequest(): int
+    {
+        return 1;
```

Comment:
Really? the don't have an API for batch sending?

Reply:
All their samples just loop sending a single message 😅 looks like they do have a bulk option through a different service ("Notify"), it looks a bit more complex than the basic SMS API used here. Will follow up tomorrow

Reply:
Added an additional `TwilioNotify` adapter 👌